pkg/templates/bindata.go

```diff
@@ -0,0 +1,249 @@
+// Code generated by go-bindata.
+// sources:
+// pkg/templates/basic-user-template.yml
+// DO NOT EDIT!
+
+package templates
```

Comment:
Do we really need to keep that generated bindata in github? Let's generate it during build and keep in git ignore.